plugin-script-python/src/main/java/io/kestra/plugin/scripts/python/package-info.java

```diff
@@ -0,0 +1,7 @@
+@PluginSubGroup(
+    description = "This sub-group of plugins contains tasks for running Python script.",
```

Comment:
```suggestion
    description = "This subgroup of plugins contains Python tasks.",
```
could potentially be clearer this way given there is both Script and Command (!=Script in theory) task